CHANGELOG.md

```diff
@@ -14,5 +14,5 @@
 
 ## 0.1.0-SNAPSHOT1
 Initial version of the EON application.
-* Based on top of [Sidechains-SDK](https://github.com/HorizenOfficial/Sidechains-SDK) version 0.7.0-SNAPSHOT
+* Based on top of [Sidechains-SDK](https://github.com/HorizenOfficial/Sidechains-SDK) version 0.8.0-SNAPSHOT
```

Comment:
let's add a new section on top instead of changing the 0.1.0-SNAPSHOT1

Reply:
Ok, fixed